hunting/windows/queries/createremotethread_by_source_process_with_low_occurrence.toml

```diff
@@ -0,0 +1,22 @@
+[hunt]
+author = "Elastic"
+integration = "logs-windows.sysmon_operational-*"
+uuid = "0545f23f-84a7-4b88-9b5b-b8cfcfdc9276"
+name = "CreateRemoteThread by source process with low occurrence"
+language = "ES|QL"
+license = "Elastic License v2"
+notes = [
+    "This hunt aggregate Sysmon CreateRemoteThread events by source process and returns the ones that we observed in only one unique host. This may indicate remote process injection.",
```

Comment:
```suggestion
    "This hunt aggregates Sysmon CreateRemoteThread events by source process and returns the ones that we observed in only one unique host. This may indicate remote process injection.",
```